bridge/client/app/_components/ktb-project-list/ktb-project-list.component.ts

```diff
@@ -1,22 +1,27 @@
-import { Component, Input } from '@angular/core';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 import { Project } from '../../_models/project';
+import { Sequence } from '../../_models/sequence';
+import { IMetadata } from '../../_interfaces/metadata';
+
+export type ProjectSequences = Record<string, Sequence[]>;
 
 @Component({
   selector: 'ktb-project-list',
   templateUrl: './ktb-project-list.component.html',
   styleUrls: ['./ktb-project-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class KtbProjectListComponent {
-  public _projects: Project[] = [];
+  @Input() metadata: IMetadata | null = null;
+  @Input() projects: Project[] = [];
+  @Input() sequences: ProjectSequences = {};
 
-  @Input()
-  get projects(): Project[] {
-    return this._projects;
+  getSequencesPerProject(project: Project): Sequence[] {
+    const latestSequences = this.sequences[project.projectName];
+    return latestSequences ?? [];
   }
 
-  set projects(value: Project[]) {
-    if (this._projects !== value) {
-      this._projects = value;
-    }
+  getShipyardversion(): string | undefined {
+    return this.metadata != null ? this.metadata?.shipyardversion : undefined;
```

Comment:
Can be simplified to `return this.metadata?.shipyardversion`. If `metadata` is `null`, `metadata?.shipyardversion` will be `undefined`. In my opinion we don't need an extra function for that and simply use `metadata?.shipyardversion` in the template